### DISCUSSION TITLE
Резьба

### DISCUSSION THREAD

**Discussion by Kosnov** [kachurovskiy/nanoels#225] [Show and tell]
Нарезка резьбы М16, шаг 2, сталь 45

https://github.com/user-attachments/assets/0f1768e9-bd72-46a4-97cb-e6dfc5de4ab0

![photo_2024-11-03_13-40-06](https://github.com/user-attachments/assets/0d7c79cf-f6e7-4023-b151-195a9ebb87ef)

Нарезка резьбы М16(трехзаходна), шаг 2, сталь 45 

https://github.com/user-attachments/assets/9a754806-c6b4-4eb5-847a-bef468942981

**Comment by PeppoMaker** [comment]
Hi @Kosnov is there a chance to share your hardware setup for that specific lather please? I will really apprettiate some suggestions. Many thanks

**Comment by Kosnov** [comment]
Hello. What exactly are you interested in?

**Reply by PeppoMaker** [reply]
Thanks for replay. I would like to understand you setup regarding encoder and motors for both axis and eventually all kind of support that you create to accommodate all the staff. I have already h4 but need some example like yours to understand how to proceed. Thanks again for your time

**Reply by PeppoMaker** [reply]
@Kosnov do you think that you have time to help me for this mod?

**Comment by Kosnov** [comment]
Приветствую. Скоро напишу какие настройки делал.

**Reply by PeppoMaker** [reply]
Hi Kosnov do you have a chance to post a picture of the motor used on both axis? Many thanks

**Comment by Kosnov** [comment]
Привет. Станка нет. Испрльзовал двигатели Nema24 и Nema34
